pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/ElasticSearchTest.java

```diff
@@ -147,7 +147,7 @@ public void searchUserByGUID() throws Exception {
             SearchResponse response = null;
             int i = 0;
             while (response == null || response.getHits().getHits().length != 0) {
-                searchSourceBuilder.query(QueryBuilders.wildcardQuery("profile.guid", "AVDDXUI451UXG3G7CAM9"));
+                searchSourceBuilder.query(QueryBuilders.wildcardQuery("profile.guid", "FCN82ETNP5WSWOSIWPR1"));
 
```

Comment:
Found some tests that I had to change the specific data that they were counting on. Here is one example. The profile with that guid was not in the database.